dev/drivers/scripts/narre/plots/jevs_narre_past31days_plots.sh

```diff
@@ -15,36 +15,10 @@ export OMP_NUM_THREADS=1
 export evs_ver=v1.0
```

Comment:
This line should not be in your driver scripts.  We cannot hardwire version numbers inside any of our scripts.  Plus, this variable is defined in run.ver and should be defined in the modulefile.  

Perry